pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## [PR Title](link_to_trello_card)
```

Comment:
Is this any different to the title on the PR when creating it in github?

Reply:
What Trello cards do we show externally?

Reply:
Good point on the title, don't think it is really necessary. 

It was kind of just to attach a link to the related Trello issue to, but we can probably instead add a "Related Links" title where we can put designs, Trello cards etc.